packages/release-mastermind/src/conditions/util/isAbandoned.ts

```diff
@@ -0,0 +1,24 @@
+import { UtilProps, UtilThis } from "..";
```

Comment:
Codacy has a fix for the issue: [The variable `UtilProps` should be named `UtilProperties`. A more descriptive name will do too.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
import { UtilProps as UtilProperties, UtilThis } from "..";
```